media_driver/media_top_cmake.cmake

```diff
@@ -21,11 +21,13 @@
 project( media )
 
 bs_set_if_undefined(LIB_NAME iHD_drv_video)
-set (LIB_NAME_EXTERNAL_LIBS "pciaccess m pthread dl rt")
+set (LIB_NAME_EXTERNAL_LIBS "pciaccess;m;pthread;dl;rt")
 
 include (FindPkgConfig)
 # find external libs, if not found, cmake will abort
-pkg_check_modules (pciaccess REQUIRED)
+foreach (LIB ${LIB_NAME_EXTERNAL_LIBS})
```

Comment:
I suggest to avoid introduction of additional variable (LIB_NAME_EXTERNAL_LIBS) and have explicit calls like:
```
pkg_check_modules(pciaccess)
pkg_check_modules(ptherad)
```
etc, This will make a script more explicit and easy to support